code/game/machinery/vending.dm

```diff
@@ -1749,6 +1752,7 @@
 	name = "Custom Vendomat"
 	desc = "A custom vending machine."
 	custom_vendor = TRUE
+	vendor_department = null
```

Comment:
Sure this won't runtime without any department?

Reply:
This is what they were before I set the default to guild